webpack/components/extensions/RegistrationCommands/fields/SetupContainerRegistryCerts.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { noop } from 'foremanReact/common/helpers';
+
+import { FormGroup, Checkbox } from '@patternfly/react-core';
+import LabelIcon from 'foremanReact/components/common/LabelIcon';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+const SetupContainerRegistryCerts = ({ value, onChange, isLoading }) => (
+  <FormGroup fieldId="reg_container_certs">
+    <Checkbox
+      ouiaId="reg-container-certs"
+      label={
+        <span>
+          {__('Set up container registry certs')}{' '}
+          <LabelIcon text={__('Set up container registry certs')} />
```

Comment:
Do we need the help text if its the same as the label text?

![image](https://github.com/user-attachments/assets/f998a22f-9716-47e0-9614-34f8401ed108)

Reply:
I had gpt suggest me a few help text versions. I like the 2nd one. Thoughts? @jeremylenz 👀 

"Set up container registry certificates to enable certificate-based authentication."

"Copy entitlement certificates to the container trust directory to enable secure access to the registry without a username or password."

"Install registry certificates to allow automatic authentication using system entitlements."

Reply:
I like the second one the best,  but it's too technical. How about something simpler:
```
Place certificates on the host, enabling access without a username or password.
```

or to be a bit more technically accurate but also more wordy:
```
Place symlinks to entitlement certificates on the host, enabling access without a username or password.
```

Reply:
![Screenshot from 2025-06-03 11-49-24](https://github.com/user-attachments/assets/b3d1bc3e-7b18-4f08-bb41-88171b63a7b0)